wiki/fr/fr-Installation-for-Linux.md

```diff
@@ -74,27 +74,27 @@ sudo make install
 
 ## Configurez votre interface audio
 
-### Configurer Jack avec QjackCtl
+### Configurer JACK avec QjackCtl
 
-Les clients Jamulus ont besoin de [Jack](https://jackaudio.org/){: target="_blank" rel="noopener noreferrer"} pour fonctionner, mais vous devez d'abord le configurer. La méthode recommandée est d'utiliser `QjackCtl`.
+Les clients Jamulus ont besoin de [JACK](https://jackaudio.org/){: target="_blank" rel="noopener noreferrer"} pour fonctionner, mais vous devez d'abord le configurer. La méthode recommandée est d'utiliser `QjackCtl`.
 
 1. Ouvrez le terminal de commande, par ex. avec Crtl-Alt-T et
-1. Exécutez la commande `qjackctl` vous verrez le **Jack Audio Connection Kit**
-2. Configurez votre interface audio comme suit (les paramètres exacts pour Jack dépendront des fonctionnalités de votre interface audio/carte son) :
+1. Exécutez la commande `qjackctl` vous verrez le **JACK Audio Connection Kit**
+2. Configurez votre interface audio comme suit (les paramètres exacts pour JACK dépendront des fonctionnalités de votre interface audio/carte son) :
 
 - Réglez l'**Interface audio** sur celle que vous souhaitez (il peut y en avoir plusieurs dans la liste)
 - Paramétrez la **Fréquence d'échantillonnage à 48000** 
 - Paramétrez les **Échantillons/Période à 128** et les Périodes/Tampons à 2 pour commencer
 
-Redémarrez Jack pour prendre en compte ces nouveaux paramètres.
+Redémarrez JACK pour prendre en compte ces nouveaux paramètres.
 
 ### Démarrer Jamulus
 
 Ouvrez Jamulus, par ex. en ligne de commande. Si vous utilisez la ligne de commande, exécutez la commande `Jamulus` (avec un « J » majuscule) qui démarre le client Jamulus.
 
 Jamulus se trouve dans `/usr/local/bin`. Vous pouvez maintenant supprimer le répertoire des sources que vous venez de compiler.
 
-Si vous avez des problèmes de son qui se coupe (en particulier les erreurs XRUN rapportées par Jack/QJackCtl), essayez de définir des valeurs plus élevées (par exemple 256 échantillons ou 3 périodes) à l'étape 3 ci-dessus. Des valeurs plus basses (par ex. 64 échantillons) pourraient apporter de meilleures performances mais peut-être plus de problèmes avec le son. Sinon, consultez la [section dépannage](Client-Troubleshooting).
+Si vous avez des problèmes de son qui se coupe (en particulier les erreurs XRUN rapportées par JACK/QJackCtl), essayez de définir des valeurs plus élevées (par exemple 256 échantillons ou 3 périodes) à l'étape 3 ci-dessus. Des valeurs plus basses (par ex. 64 échantillons) pourraient apporter de meilleures performances mais peut-être plus de problèmes avec le son. Sinon, consultez la [section dépannage](Client-Troubleshooting).
```

Comment:
I think we also discussed to use "QjackCtl" not "QJackCtl"

Reply:
This PR is about jack/Jack/JACK.
I've got a bunch of other consistency PR to make, but I prefer to look at these one by one.
Please, merge this one as is.

Reply:
Ah. Ok

Reply:
thank you very much